spec/design_topics/data_interchange.rst

```diff
@@ -40,7 +40,7 @@ The interchange mechanism must offer the following:
    ABI is required for packages to work well together.*
 
 The best candidate for this protocol is
```

Comment:
```suggestion

DLPack Support
--------------

The best candidate for this protocol is
```

@leofang How about we add the section up here? It highlights that DLPack is the chosen standard. Maybe, we can change the name to something like `Choosing DLPack` or `DLPack: An in-memory tensor structure`

Reply:
Yup that'd work too!